lisa/spine.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+# Written by Lisa Eunyoung Lee - Jun 2022
+# Revised by Lisa Eunyoung Lee - May 2024
```

Comment:
You can keep the author, but I suggest removing the dates of modification, because git takes care of that

Reply:
fixed in https://github.com/ivadomed/canproco/pull/90/commits/517ccdd69696cd696fd41641a653e4c159d43f28